This is just workaround. It should be removed after proper fixing in upstream.
Patch fixes problem with lmdb pkg config. It also adds workaround for get_option
which is now not propagated in OpenWrt meson host package.
--- a/meson.build
+++ b/meson.build
@@ -23,10 +23,10 @@ libknot = dependency('libknot', version:
 libdnssec = dependency('libdnssec', version: knot_version)
 libzscanner = dependency('libzscanner', version: knot_version)
 libuv = dependency('libuv', version: '>=1.7')
-lmdb = dependency('lmdb', required: false)
-if not lmdb.found()  # darwin workaround: missing pkgconfig
-  lmdb = meson.get_compiler('c').find_library('lmdb')
-endif
+##lmdb = dependency('lmdb', required: false)
+##if not lmdb.found()  # darwin workaround: missing pkgconfig
+lmdb = meson.get_compiler('c').find_library('lmdb')
+##endif
 gnutls = dependency('gnutls')
 luajit = dependency('luajit')
 # NOTE avoid using link_args for luajit due to a macOS issue
